third_party/rust/tracing/v0_1/crate/Cargo.toml

```diff
@@ -98,16 +95,20 @@ optional = true
 version = "0.2.9"
 
 [dependencies.tracing-attributes]
-version = "0.1.23"
+version = "0.1.27"
```

Comment:
Looks like you might need [tracing-attributes 0.1.24](https://github.com/tokio-rs/tracing/blob/tracing-attributes-0.1.27/tracing-attributes/CHANGELOG.md#0124-april-24th-2023) or later as well.

Reply:
Oh, you did that already. Nevermind.